hmda-data-publisher/src/main/scala/hmda/publisher/helper/QuarterTimeBarrier.scala

```diff
@@ -52,10 +56,13 @@ object QuarterTimeBarrier {
       case Period.y2020Q3 => LocalDate.ofYearDay(2020,rulesConfig.qf.q3.endDayOfYear)
       case Period.y2021Q1 => LocalDate.ofYearDay(2021,rulesConfig.qf.q1.endDayOfYear)
       case Period.y2021Q2 => LocalDate.ofYearDay(2021,rulesConfig.qf.q2.endDayOfYear)
-      case Period.y2021Q3 => LocalDate.ofYearDay(2021,rulesConfig.qf.q3.endDayOfYear)
+      case Period.y2021Q3 => LocalDate.ofYearDay(2021,rulesConfig.qf.q3.endDayOfYear - 1)
```

Comment:
Now that we use the actual year, you should be able to remove the "-1" and still get the tests to pass